pygmt/src/dimfilter.py

```diff
@@ -0,0 +1,143 @@
+"""
+dimfilter - Filter a grid file by dividing the filter circle.
```

Comment:
Is this short description accurate?

Reply:
fixed in 87c7cb6; I must have left it in when copying another file